src/model/CoilHeatingDXSingleSpeed.cpp

```diff
@@ -1052,11 +1069,27 @@ namespace model {
   }
 
   double CoilHeatingDXSingleSpeed::ratedSupplyFanPowerPerVolumeFlowRate() const {
-    return getImpl<detail::CoilHeatingDXSingleSpeed_Impl>()->ratedSupplyFanPowerPerVolumeFlowRate();
+    return getImpl<detail::CoilHeatingDXSingleSpeed_Impl>()->ratedSupplyFanPowerPerVolumeFlowRate2017();
```

Comment:
I would add a Warning message here (and on aall deprecated methods)

```
LOG(Warn, "As of 3.5.0, this method is deprecated. Use CoilHeatingDXSingleSpeed::ratedSupplyFanPowerPerVolumeFlowRate2017() instead. It will be removed in three releases.")
```

or something like that. We also want to mention the deprecation in the Release Notes